### PR TITLE
Added a "Game Of Life" implementation to /raylib

### PR DESCRIPTION
**Pull request by tsillus**

### CODE REVIEW

**Comment by Kelimion** [comment]
That's a really fun example, thanks.

**Comment by tsillus** [comment]
This PR is now half a year old. Is there anything I should improve about this example, so it can get accepted?

**Comment by Kelimion** [comment]
> This PR is now half a year old. Is there anything I should improve about this example, so it can get accepted?

I suggested a number of changes, none of which have been made yet. That's what's been holding up acceptance.

From this:
image
Down to this:
image
And a few more things as well.

**Comment by tsillus** [comment]
I don't see those suggestions. Where do I find them?

**Comment by Kelimion** [comment]
> I don't see those suggestions. Where do I find them?

Literally on this very page. If you don't see them your browser is broken.

**Comment by Kelimion** [comment]
> I don't see them either on Firefox or Chrome.
What about the [Changed Files](https://github.com/odin-lang/examples/pull/24/files) tab? They're inlined there as well.

**Comment by Kelimion** [comment]
> @Kelimion Looking at your screenshots it looks like your review may not have been submitted. I don't see them either.
Well that's bad UX. Thanks.

**Comment by Kelimion** [comment]
So as it turns out I reviewed it on the day of submission but GH decided to have poor UX and not show any of it, because why not?

File: raylib/game_of_life/game_of_life.odin [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+Window :: struct { 
+    name          : cstring,
+    width         : i32, 
+    height        : i32,
+    fps           : i32,
+    control_flags : rl.ConfigFlags,
+}
+
+Game :: struct {
```
[the remark this answers is not in the document]
Comment (tsillus) [reply raylib/game_of_life/game_of_life.odin:33]: Done.

File: raylib/game_of_life/game_of_life.odin [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+import fmt  "core:fmt"
+import time "core:time"
+import rl   "vendor:raylib"
+
+
+Window :: struct { 
+    name          : cstring,
+    width         : i32, 
+    height        : i32,
+    fps           : i32,
```
[the remark this answers is not in the document]
Comment (tsillus) [reply raylib/game_of_life/game_of_life.odin]: window.fps is actually set to `60` in line 171:

`window := Window{"Game Of Life", 1024, 1024, 60, rl.ConfigFlags{ .WINDOW_RESIZABLE }}`

File: raylib/game_of_life/game_of_life.odin [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    height :  i32,
+    alive  : []u8,
+}
+
+Cell :: struct { 
+    width  : f32,
+    height : f32,
+}
+
+UserInput :: struct {
```
[the remark this answers is not in the document]
Comment (tsillus) [reply raylib/game_of_life/game_of_life.odin]: Done.

File: raylib/game_of_life/game_of_life.odin [hunk 4]
```diff
@@ -0,0 +1,254 @@
+package game_of_life;
```
[the remark this answers is not in the document]
Comment (tsillus) [reply raylib/game_of_life/game_of_life.odin]: Done.

File: raylib/game_of_life/game_of_life.odin [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+ * Right-Click to kill a cell
+ * Press <Space> to (un)pause the game
+ * Press <Esc> to close the game
+
+ The game starts paused. 
+
+**********************************************************************/
+
+
+import fmt  "core:fmt"
```
[the remark this answers is not in the document]
Comment (tsillus) [reply raylib/game_of_life/game_of_life.odin]: removed.

**Pull request merged**